cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -216,7 +216,7 @@ impl Node<Option<cst::Policy>> {
         id: ast::PolicyID,
         errs: &mut ParseErrors,
     ) -> Option<ast::Template> {
-        let (maybe_policy, src) = self.as_inner_pair();
+        let maybe_policy = self.as_inner();
```

Comment:
I might go a step further and inline this with `let policy = ...` and do the same for similar uses in other functions.

Reply:
Planning on that as a followup